styles/index_style.css

```
@@ -0,0 +1,102 @@
+/* 
+Dark Red: #8F0C0C
+Rosewood: #5F1212 
+Dark Sienna: #230707
+Tuscany: #CAA2A2
+Snow: #F7F2F2
+*/
+
+* {
+    margin: 0;
+    padding: 0;
+}
```

This eliminates all of the default spacing on the page (including between paragraphs).  It's better to apply this to specific elements that are giving you trouble (like `html` and `body`, for example):

```suggestion
html, body {
    margin: 0;
    padding: 0;
}
```